src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxNumber = 100;

```

Comment:
[Don't forget how to name constants according to style guide (the same with naming of enum values)](https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#dont-forget-how-to-name-constants-according-to-style-guide-the-same-with-naming-of-enum-values)